tasks/eva_3417/src/main/groovy/eva3417/RemediateLowerCaseRefAlt.groovy

```diff
@@ -0,0 +1,270 @@
+package eva3417
+
+
+import groovy.cli.picocli.CliBuilder
+import org.slf4j.LoggerFactory
+import org.springframework.batch.core.JobParameter
+import org.springframework.batch.core.JobParameters
+import org.springframework.batch.core.StepExecutionListener
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory
+import org.springframework.batch.core.launch.JobLauncher
+import org.springframework.batch.core.launch.support.RunIdIncrementer
+import org.springframework.batch.core.repository.JobRepository
+import org.springframework.batch.item.*
+import org.springframework.batch.repeat.policy.SimpleCompletionPolicy
+import org.springframework.data.mongodb.core.BulkOperations
+import org.springframework.data.mongodb.core.query.Criteria
+import org.springframework.data.mongodb.core.query.Query
+import org.springframework.data.mongodb.core.query.Update
+import org.springframework.transaction.PlatformTransactionManager
+import uk.ac.ebi.ampt2d.commons.accession.hashing.SHA1HashingFunction
+import uk.ac.ebi.eva.accession.core.batch.io.SubmittedVariantDeprecationWriter
+import uk.ac.ebi.eva.accession.core.model.ISubmittedVariant
+import uk.ac.ebi.eva.accession.core.model.SubmittedVariant
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantEntity
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantInactiveEntity
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantOperationEntity
+import uk.ac.ebi.eva.accession.core.summary.SubmittedVariantSummaryFunction
+import uk.ac.ebi.eva.accession.deprecate.Application
+import uk.ac.ebi.eva.accession.deprecate.batch.listeners.DeprecationStepProgressListener
+import uk.ac.ebi.eva.accession.deprecate.parameters.InputParameters
+import uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment
+import uk.ac.ebi.eva.groovy.commons.RetryableBatchingCursor
+import uk.ac.ebi.eva.metrics.metric.MetricCompute
+
+import java.text.ParseException
+import java.time.LocalDateTime
+import java.util.function.Function
+import java.util.stream.Collectors
+
+import static org.springframework.data.mongodb.core.query.Criteria.where
+import static org.springframework.data.mongodb.core.query.Query.query
+import static uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment.*
+
+def cli = new CliBuilder()
+cli.prodPropertiesFile(args: 1, "Production properties file for connecting to prod db", required: true)
```

Comment:
Do I understand this right that you can specify a dev property file here and that would read and write to/from dev ?

Reply:
Yes, we can pass a dev properties file to read/wrtite from dev. Have updated the parameter name to avoid confusion